src/components/Calculator.jsx

```diff
@@ -18,11 +18,16 @@ const Calculator = () => {
   };
 
   return (
-    <div className="calculator">
-      <OutputScreen state={calculator} />
-      <OperatorKeys handleEvent={handleEvent} />
-      <NumericKeys handleEvent={handleEvent} />
-    </div>
+    <main className="calculator-container">
+      <header>
+        <h2>Lets do some math!</h2>
+      </header>
```

Comment:
- [optional] your app looks perfect. It will be nice if you add some hover effects on the calculator buttons, so it indicates which one has been clicked

Reply:
@tufoinnkuo10 If you mean the effect when the button is clicked it's already there please check clearly.

Reply:
@AbelG101 ok then, but guess it's not clear enough. You can just adjust it a little.. Nice work 👏🏾